src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,49 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 100;
+    private static final int MIN_LENGTH = 6;
+    private static final int MAX_LENGTH = 20;
```

Comment:
as described in common mistakes: 
<img width="987" alt="image" src="https://github.com/mate-academy/jv-registration-validation/assets/53354844/a7dc5452-39bf-4553-b449-7441c38edb9d">